package.json

```diff
@@ -17,6 +17,8 @@
   },
   "homepage": "https://github.com/Zubenna/restaurant-page#readme",
   "devDependencies": {
+    "css-loader": "^5.0.1",
+    "style-loader": "^2.0.0",
```

Comment:
- [ ] Kindly include the necessary run commands in the scripts section of package.json.

````
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

````

Reply:
- Also if you'll be using the start script you'll need to include the webpack-dev-server package.-